### PR TITLE
rename the http folder to a better name, than python

### PR DESCRIPTION
**Pull request by matzew**
# Changes

- as per title
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Comment by lkingland** [comment]
I don't think this will quite work as intended.

I might be wrong, but this package is already published on PyPI as "func-python" (and imported as "func_python".  By changing this, we would need to duplicate all our work and publish two separate packages; one for each of "http" and "cloudevents".

Furthermore, it is my understanding the "Pythonic" idiom is to use files as modules, rather than directories whic is idiomatic in Go.  Thuse in Python one would expect an `http.py` and a `cloudevent.py` to implement the HTTP and CloudEvent modules, respectively.

This alignment 1:1 from published package name to import name is the reason the current codebase does not require "packages" be explicitly defined in the pyproject.toml.

Thus, I am fairly sure the current file hierarchy is correct, and the simplest option; our only  needing to add a "cloudevents.py" file right beside "http.py".

I am happy to entertain proof otherwise, because I am not a Python packaging expert!

File: pyproject.toml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 name = "func-python"
 version = "0.2.0"
 description = "Knative Functions Python Middleware"
+packages = [
+    { include = "func_http", from = "src" },
+]
+
```
Comment (lkingland) [inline pyproject.toml:8]: The reason the folder is named `func_python` is such that it matches the package name as published on PyPI `func-python`.  Hence the reason there's no need for an explicit `packages` section in the pyproject.toml

File: tests/test_http.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import signal
 import threading
 import time
-from func_python.http import serve
+from func_http.http import serve
```
Comment (lkingland) [inline tests/test_http.py:7]: With an `http.py` and a `cloudevents.py` file in the `func_python` directory, the two imports would be:
`from func_python.http import serve` for HTTP  middleware and
`from func_python.cloudevents import serve` for CloudEVent middleware
That seems correct to me

**Comment by matzew** [comment]
/close

thanks for the insights. Agree not needed

File: cmd/fhttp/main.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import argparse
 import logging
-from func_python.http import serve
+from func_http.http import serve
```
Comment (matzew) [inline cmd/fhttp/main.py:3]: looking _now_ at this I see that `func_http.http` is a bit unfortunate as well :sweat_smile:

**Pull request closed without merging**